### ISSUE TITLE
Proposed provisioner: AWS Fargate

### ISSUE THREAD

**Issue by aidansteele** [inlets/inletsctl#51]
This one’s a bit different, but I thought it would be worth discussing. Fargate allows for public IP addresses and seems like it could meet all the requirements laid out for a new provisioner. What do you think? My main motivation is that I imagine it starts up quicker than other options.

**Comment by alexellis** [comment]
Imho Fargate is very slow to boot and expensive vs. EC2. If you can find some numbers to prove otherwise we may take a look.

**Comment by aidansteele** [comment]
Good point on the numbers. I’ll try come up with some and report back. If it’s indeed no faster, it makes it a much easier “no”
